.jenkins/jenkins.sh

```diff
@@ -142,12 +142,11 @@ export FV3_STENCIL_REBUILD_FLAG=False
 export TEST_DATA_HOST="${TEST_DATA_DIR}/${experiment}/"
 export EXPERIMENT=${experiment}
 if [ -z ${JENKINS_TAG} ]; then
-    if [ ${#JOB_NAME} -gt 85 ]; then
-	NAME=`echo ${JOB_NAME} | md5sum | cut -f1 -d" "`
-    else
-	NAME=${JOB_NAME}
-    fi
-    export JENKINS_TAG=${NAME//[,=\/]/-}-${BUILD_NUMBER}
+    export JENKINS_TAG=${JOB_NAME//[,=\/]/-}-${BUILD_NUMBER}
```

Comment:
Since I'm only marginal at scripting, can you quickly explain what the `//[,=\/]/-` does?

Reply:
We want the jenkins tag to be a unique identifier for the test case being run in case another job is running on the same machine, that they don't accidentally use the wrong image. docker tags can only have a limited character set ... but we want a tag that is based on the name of the project, JOB_NAME is the closest thing to what we want. Except, when using a multiconfiguration project (e.g. run the same plan on different machines and/or with different arguments), the JOB_NAME gets characters ',', '=' and '/', that are illegal as a tag.

Reply:
Oh you mean for scripting -- that does a find replace of those characters with a '-'

Reply:
should be replacing all `=` and `/` by `-`, I think.